### PR TITLE
added new parameter disable_notifications in relation to setenv.sh

### PR DESCRIPTION
**Pull request by f3rr**
in some cases its needed to comment in DISABLE_NOTIFICATIONS in setenv.sh. this pull request will add this function to this module.

### CODE REVIEW

**Comment by mkrakowitzer** [comment]
Thanks f3rr, would you consider adding a test?

**Comment by f3rr** [comment]
Hey Mkrakowitzer, Tests added

File: spec/classes/jira_config_spec.rb [hunk 1]
```diff
@@ -171,6 +172,15 @@
           it { should contain_file('/opt/jira/atlassian-jira-6.3.4a-standalone/conf/context.xml')
             .with_content(/<Resource name = "testdb"\n        auth = "Container"\n    \/>/) }
         end
+
+        context 'disable notifications' do
+          let(:params) {{
+            :version  => '6.3.4a',
+            :javahome => '/opt/java',
```
Comment (mkrakowitzer) [inline spec/classes/jira_config_spec.rb]: Here we should specify the 'disable_notifications => true',

File: spec/classes/jira_config_spec.rb [hunk 2]
```diff
@@ -171,6 +172,15 @@
...
+          }}
+          it { should contain_file('/opt/jira/atlassian-jira-6.3.4a-standalone/bin/setenv.sh')
+            .with_content(Regexp.new('^#DISABLE_NOTIFICATIONS=')) }
```
Comment (mkrakowitzer) [inline spec/classes/jira_config_spec.rb]: This regex will need to be updated. Something like /^DISABLE_NOTIFICATIONS=/. 

You don't need to use Regexp.new, You can just use /match/.

File: spec/classes/jira_config_spec.rb (hunk 1 above)
Comment (f3rr) [inline spec/classes/jira_config_spec.rb]: sorry, my fault. Its fixed now. I hope its ok now.

**Comment by mkrakowitzer** [comment]
:+1: thanks!

**Pull request merged**